include/dpp/channel.h

```diff
@@ -862,6 +870,9 @@ struct active_thread_info {
  * @brief A map of threads alongside optionally the thread_member tied to the bot if it is in the thread. The map's key is the thread id. Returned from the cluster::threads_get_active method
  */
 using active_threads = std::map<snowflake, active_thread_info>;
+/**
+ * @brief A stream wrapper to send messages
```

Comment:
What does this document? The rest of the block under it is empty.

Reply:
Whoops! I originally had the streams for channels and users in their respective headers, and I forgot to remove the docs when I removed the actual declarations